app/assets/javascripts/app/phone-internationalization.js

```diff
@@ -0,0 +1,57 @@
+import { PhoneFormatter } from 'field-kit';
+
+const I18n = window.LoginGov.I18n;
+const phoneFormatter = new PhoneFormatter();
+
+const PHONE_UNSUPPORTED_AREA_CODES = {
+  648: 'American Samoa',
```

Comment:
Also I think we should move these values to something on the server, and expose them as `data-` attributes in the DOM or something (that way we can double check these values on the server which also works for clients without JS)